lua/nvim-tree/view.lua

```diff
@@ -3,7 +3,11 @@ local a = vim.api
 local M = {}
 
 function M.nvim_tree_callback(callback_name)
-  return string.format(":lua require'nvim-tree'.on_keypress('%s')<CR>", callback_name)
+  info = {
+    name = callback_name,
+    callback = string.format(":lua require'nvim-tree'.on_keypress('%s')<CR>", callback_name),
+  }
+  return info.callback
```

Comment:
not sure what this change is made for ?

Reply:
Sorry, that was from a previous though process 😅
I've removed it now